Drop after the respective upstream fix.

From e3f22abab59aad3af3b555c4aa9af4441d7b4085 Mon Sep 17 00:00:00 2001
From: Daniel Kolesa <daniel@octaforge.org>
Date: Thu, 17 Dec 2020 22:28:12 +0100
Subject: [PATCH] powerpc: only call cleanup_cpu_mmu_context on PPC64 powermacs

Originally introduced in 01b0f0eae0812e80efeee4ee17687e5386335e08.

This was only implemented on book3s64, so on ppc32 build fails
with implicit function declaration.

Signed-off-by: Daniel Kolesa <daniel@octaforge.org>
---
 arch/powerpc/platforms/powermac/smp.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/powerpc/platforms/powermac/smp.c b/arch/powerpc/platforms/powermac/smp.c
index adae2a67..7ad90fd0 100644
--- a/arch/powerpc/platforms/powermac/smp.c
+++ b/arch/powerpc/platforms/powermac/smp.c
@@ -911,7 +911,9 @@ static int smp_core99_cpu_disable(void)
 
 	mpic_cpu_set_priority(0xf);
 
+#ifdef CONFIG_PPC64
 	cleanup_cpu_mmu_context();
+#endif
 
 	return 0;
 }
-- 
2.29.2

